File: core/client/game/src/commonGame/net/base/LoginHttpRequest.cs
using ShineEngine;

/// <summary>
/// 中心服http请求
/// </summary>
public abstract class LoginHttpRequest:BytesHttpRequest
{
	public LoginHttpRequest()
	{
		setNeedFullWrite(ShineSetting.clientMessageUseFull);
		_url=LocalSetting.loginHttpURL + "/" + ShineSetting.bytesHttpCmd;
	}
}

File: core/clientLaya/game/src/commonGame/net/base/LoginHttpRequest.ts
namespace Shine
{
	export abstract class LoginHttpRequest extends BytesHttpRequest
	{
	    constructor()
		{
			super();
			this.setNeedFullWrite(ShineSetting.clientMessageUseFull);
			this._url=LocalSetting.loginHttpURL + "/" + ShineSetting.bytesHttpCmd;
		}
	}
}